packages/cli/README.md

```diff
@@ -8,6 +8,12 @@
 
 ---
 
+|                  📊 Getting Started                  |               🌐 Portal Integration                |              🛠️ CI Automation              |
+| :--------------------------------------------------: | :------------------------------------------------: | :----------------------------------------: |
+| **[How to setup](#getting-started)** a basic project | Sort, filter **[your goals](#portal-integration)** | Updates **[on every PR](#ci-integration)** |
```

Comment:
Broken link:

```suggestion
| **[How to setup](#getting-started)** a basic project | Sort, filter **[your goals](#portal-integration)** | Updates **[on every PR](#-ci-automation)** |
```

Testable here: https://github.com/code-pushup/cli/blob/docs-cli-improvements/packages/cli/README.md#-ci-automation

Reply:
Thx for the testing tipp